src/views/SpaceCreate.vue

```diff
@@ -345,12 +342,9 @@ function handleOsnapToggle() {
 }
 
 onMounted(async () => {
-  const network = props?.space?.plugins?.safeSnap?.safes?.[0]?.network;
-  const umaAddress = props?.space?.plugins?.safeSnap?.safes?.[0]?.umaAddress;
-  if (network && umaAddress) {
-    // this is how we check if osnap is enabled and valid.
-    osnap.value.enabled =
-      (await safeSnapPlugin.validateUmaModule(network, umaAddress)) === 'uma';
```

Comment:
wondering if this might break the old plugin?

Reply:
yeah we can leave this as is